src/app.ts

```diff
@@ -1,6 +1,87 @@
 
 const startApp = async () => {
+    const tbody = document.querySelector('[data-sink]')
+    const tr = tbody?.querySelectorAll('tr')
+    const prevBtn = document.querySelector('[data-prevbtn')
+    const nextBtn = document.querySelector('[data-nextbtn')
```

Comment:
Codacy has a fix for the issue: [Replace `····const·nextBtn·=·document.querySelector('[data-nextbtn')` with `··const·nextBtn·=·document.querySelector("[data-nextbtn");`](https://app.codacy.com/gh/alic-xc/Test/pullRequest?prid=9965044)
```suggestion
  const nextBtn = document.querySelector("[data-nextbtn");
```